bench/config.toml

```diff
@@ -24,9 +24,14 @@ convert = false
 cmd.flatgfa = '{fgfa} -I {files[gfa]}'
 cmd.slow_odgi = '{slow_odgi} norm {files[gfa]}'
 cmd.odgi = '{odgi} view -g -i {files[gfa]}'
-cmd.gfatools = "{gfatools} view {files[gfa]}"
+cmd.gfatools = '{gfatools} view {files[gfa]}'
 
 [modes.depth]
 cmd.flatgfa = '{fgfa} -i {files[flatgfa]} depth'
 cmd.odgi = '{odgi} depth -i {files[og]} -d'
-cmd.slow_odgi = '{slow_odgi} depth {files[gfa]}'
\ No newline at end of file
+cmd.slow_odgi = '{slow_odgi} depth {files[gfa]}'
+
+[modes.chop]
+cmd.flatgfa = '{fgfa} -i {files[flatgfa]} chop -c 3'
+cmd.odgi = '{odgi} chop -i {files[og]} -c 3 -o - | {odgi} view -g -i - | {slow_odgi} norm --nl'
```

Comment:
For benchmarking purposes, I think we should probably omit the `view` and `norm` steps. This makes for a fairer comparison, where we just measure the time taken to do the data structure transformation and not the time to print out the results. (This of course means that we can't simultaneously check correctness, but I suppose that is probably best done separately.)

Reply:
Yep, you're absolutely right. This explains the weird benchmarking results that I was getting for `chop`. Thanks for catching it!